jobs/awslogs/templates/bin/additional-logs.sh

```diff
@@ -17,3 +17,4 @@ do
   echo "initial_position = start_of_file"
   echo "log_group_name = $GROUP_NAME"
 done > /var/vcap/jobs/awslogs/conf.d/all-vcap-logs.conf
+monit restart awslogs
```

Comment:
Does the awslogs agent taken a signal to reload the config?  If so can we do something like
    
     kill -HUP `cat /var/vcap/sys/run/awslogs/awslogs.pid` 

here instead of stopping/starting the daemon entirely?